Sources/Router/Colorization/OAColoringType.mm

```diff
@@ -160,6 +160,21 @@ - (BOOL) isGradient
     return self == self.class.SPEED || self == self.class.ALTITUDE || self == self.class.SLOPE;
 }
 
+- (BOOL) isSlope
```

Comment:
This class was ported by me from Android, there are no such methods there, so they shouldn't be here either.